src/Functions/polyArcLength3.m

```diff
@@ -1,21 +1,26 @@
-%% Calculate the arc length of an interval on a parameterized polynomial in R3
-% Params:
-%     polyCoeff - coefficient matrix
-%     lowerLimit
-%     upperLimit
+function arcLength = polyArcLength3(polyCoeff, lowerLimit, upperLimit)
 %
-% lowerLimit / upperLimit parameters can take scalar as well as vector inputs,
-% in the vector case arc lengths are returend for all pairs of intervals
-% between lowerLimit and upperLimit
+% USAGE:
 %
-% Andreas Husch
-% Centre Hospitalier de Luxembourg / Luxembourg Centre for Systems
-% Biomedicine, University of Luxembourg
-% 2016  - 2017
-% mail@andreashusch.de, husch.andreas@chl.lu
-
+%    arcLength = polyArcLength3(polyCoeff, lowerLimit, upperLimit)
+%
+% INPUTS: 
+%    polyCoeff:     Coefficient matrix
```

Comment:
@adhusch Could you please confirm that the descriptions of the inputs and outputs are correct?

Reply:
They are perfect :)

Reply:
Thank you very much, Andreas!